python/tests/kat/t_mappingtests.py

```diff
@@ -305,7 +305,7 @@ def config(self):
 hostname: "*"
 prefix: /{self.name}/status/
 rewrite: /status/
-service: httpbin.default
+service: httpbin.plain-namespace
```

Comment:
Huh. Why is this necessary?

Reply:
There's a question for @acookin

Reply:
because that's the naemsapce that the actual service gets deployed to. default was only working because another test was creating it